lib/registration.pm

```diff
@@ -776,7 +776,7 @@ sub registration_bootloader_cmdline {
     set_var('SCC_URL', 'https://scc.suse.com') unless get_var('SCC_URL');
```

Comment:
In fact, this line will cause it must go in the if thread, so if no special reason could we remove it? Or change the logic to set regurl in correct condition( SCC_URL or SMT_URL set)?